.github/workflows/lint.yml

```
@@ -0,0 +1,43 @@
+name: fmt
+
+on:
+  push:
+    branches:
+      - "feature/**"
+
+jobs:
+  format:
+    name: run lint:fix
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - uses: actions/cache@v3
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+              ${{ runner.os }}-yarn-
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile 

```

--frozen-lockfileは非推奨

```suggestion
        run: yarn install --immutable

```

---

[b74db77](https://github.com/suzuka-kosen-festa/2022-server/pull/17/commits/b74db7786a650752c3aee53763a2b87cfc42a70e)